deltachat-ios/Controller/WebxdcViewController.swift

```diff
@@ -227,10 +211,16 @@ class WebxdcViewController: WebViewViewController {
             }
         }
     }
-    
-    private func updateWebxdc(statusId: Int) {
-        let statusUpdates = self.dcContext.getWebxdcStatusUpdates(msgId: messageId, statusUpdateId: statusId)
-        logger.debug("status updates: \(statusUpdates)")
+
+    var lastSerial: Int = 0
+    private func updateWebxdc() {
+        let statusUpdates = dcContext.getWebxdcStatusUpdates(msgId: messageId, lastKnownSerial: lastSerial)
+        if let data: Data = statusUpdates.data(using: .utf8),
+           let array = (try? JSONSerialization.jsonObject(with: data, options: [])) as? [Any],
+           let first = array.first as? [String: Any],
+           let maxSerial = first["max_serial"] as? Int {
+            lastSerial = maxSerial
```

Comment:
@cyBerta here i also stumbled upon the `if let` thingie :) maybe we should take care that the `let` are really horizontally aligned, then it is at least a bit better readable - and swift does not complain and we still follow swift style.